lib/inc/internal/nodes/config_node_simple_value.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "abstract_config_node.hpp"
+#include <internal/values/abstract_config_value.hpp>
+
+namespace hocon {
+
+    class config_node_simple_value : public abstract_config_node {
+    public:
+        config_node_simple_value(shared_token value);
+
+        shared_token get_token() const;
+        std::unique_ptr<abstract_config_value> const& get_value() const;
```

Comment:
With unique_ptr, it makes more sense to just return a const& of the object. unique_ptr isn't copyable, so there's no useful reason to have access to a `const&` of it.